### ISSUE TITLE
Add. 14,523, foll. 47–48 | https://bl.vuexistapps.us/manuscript/231 (Part9)

### ISSUE THREAD

**Issue by Erfan-Syriaca** [srophe/britishLibrary-data#1082]
### Problems with Manuscript Overview

- Please check if Wright's classification should be "Prayers" or "Hymns"

### Categories of Manuscript Overview Problems

- [ ] Whitespace
- [ ] Typo(s)
- [ ] Incorrect content
- [ ] Missing content

### Problems with Physical Description

- Shouldn't the following sentence be placed in Hand 1? "from a neatly written manuscript of the xith cent."

### Categories of Physical Description Problems

- [ ] Whitespace
- [ ] Typo(s)
- [X] Incorrect content
- [ ] Missing content

**Comment by mikey-tang16** [comment]
Resolved:
- handnote fixed

Question
- Should we change the classification from prayers to hymns? @wlpotter

**Comment by wlpotter** [comment]
Yes, please update this (it may be found in the `msPart/head/listRelation`). The one for "Service Books" should stay, though, since this is also the broader category for Hymns

**Comment by mikey-tang16** [comment]
Subject classification changed to "hymns"
